### DISCUSSION TITLE
Tomb: Stone port for Red Eclipse 2

### DISCUSSION THREAD

**Discussion by MirceaKitsune** [redeclipse/base#1454] [Maps and Content Showcase]
I'm happy to offer my latest map port for Red Eclipse 2.1, this time of the famous map Stone by Wicked. I've replaced the missing textures, redid the weapon and entity setup, and most importantly added several new areas most notably a playable exterior at the top! The remix is modernized but still faithful to the original, now themed to be an Egyptian map with bright days and starry nights.

- [r528](https://github.com/redeclipse/base/files/14060039/tomb_r528.zip) Initial version. Fully playable and without known issues, more tweaks and details likely to be added in the future.
- [r563](https://github.com/redeclipse/base/files/14091506/tomb_r563.zip) Fixed a few issues and added missing details.
- [r666](https://github.com/redeclipse/base/files/14133127/tomb_r666.zip) Yes this version actually happened, don't worry it's not cursed. Fixed several issues found by other players and developers as well as other corrections based on their feedback, most notably holes in the clips surrounding the exterior walls which now use pushers to keep the player away. Some texturing improvements and extra details were added such as dust particles to make wind direction clearer.

![20240125_035040](https://github.com/redeclipse/base/assets/825418/fca0096e-63ce-40f7-937a-daf665aa7c21)
![20240125_035130](https://github.com/redeclipse/base/assets/825418/a3ea88b3-77d6-4dbe-8c53-ddd54e920bdb)
![20240125_035612](https://github.com/redeclipse/base/assets/825418/40374650-5a05-42b9-b369-723d8b4e05fe)
![20240125_035646](https://github.com/redeclipse/base/assets/825418/e2430ebd-4324-40f7-8550-f2fa0f24d402)
![20240125_062411](https://github.com/redeclipse/base/assets/825418/a9647a13-e9d0-4807-b27e-66fb9e7b4854)
![20240125_062941](https://github.com/redeclipse/base/assets/825418/9e8ee807-9488-41b1-a650-4544ccbe4a23)
![20240125_164332](https://github.com/redeclipse/base/assets/825418/9245a98e-ffc7-4245-9de2-2db92868e898)
![20240125_172523](https://github.com/redeclipse/base/assets/825418/3659d79b-405e-468c-8b2e-0c3be7496cc6)

**Comment by Calinou** [comment]
This looks pretty nice, but I'd advise reducing the daytime fog a bit as it gets quite heavy especially indoors. If reducing the rifle's viability is what you want to achieve, I'd suggest additing more cover options to the map instead.

I like the egyptian theme, it's good to see a new map that isn't set in a futuristic/sci-fi universe for once :slightly_smiling_face:

**Reply by MirceaKitsune** [reply]
Thanks for the feedback. The fog was to keep the air a bit more dense and claustrophobic indoor, I'll reduce it a little in the next version then.
